src/PositionManager.sol

```diff
@@ -327,6 +327,8 @@ contract PositionManager is
             (liquidityDelta - feesAccrued).validateMinOut(amount0Min, amount1Min);
         }
 
+        if (positionConfigs[tokenId].hasSubscriber()) _unsubscribe(tokenId, config);
```

Comment:
this somewhat inefficiently must be doing an SSTORE of
positionConfig.isSubscribed = false

and then because it burns it then just deletes the position config anyway?

Reply:
but probably much neater code to keep it like this